### ISSUE TITLE
how to get the width of a space in a particular font-size?

### ISSUE THREAD

**Issue by dhowe** [nodebox/opentype.js#125]
not sure if this is a bug, but glyph.getMetrics() for a space returns:
````
{xMin: Infinity, yMin: Infinity, xMax: -Infinity, yMax: -Infinity, leftSideBearing: 0, rightSideBearing: Infinity}
````

currently I'm using the following to get the width:
````javascript
myFont.charToGlyph(' ').advanceWidth * ((1 / myFont.unitsPerEm) * fontSize)

**Comment by fdb** [comment]
Hmm that's not good. The metrics are calculated from the glyph outlines, but since there are no outlines the glyph is considered to be infinitely big.

The advanceWidth is a good solution for calculating spacing between glyphs, though.

**Comment by dhowe** [comment]
This has been working for me until just now, when I encountered a font for which the 'space' glyph does not have an advanceWidth property:

![opent](https://cloud.githubusercontent.com/assets/737638/9747975/d41037d4-56b3-11e5-9e08-61cab3fabb45.png)

**Comment by dhowe** [comment]
Any chance of an update to get the right metrics for a space (we are still special-casing it in our app)?

**Comment by fdb** [comment]
Can you send me the font that doesn't have the advanceWidth value for space? You can email it to me directly (frederik@debleser.be) if it's not freely available.

I'll see if I can update the getMetrics code, in any case.

**Comment by fdb** [comment]
@dhowe This should be fixed in release 0.4.11.

**Comment by dhowe** [comment]
excellent, thanks...
